ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -290,90 +290,81 @@ class ShiftViewController: BaseViewController {
     }
     
     func validationMessage() -> String {
-        var message: String = ""
-        guard let model = self.model else { return message }
-        var counter = 1
+        var messages: [String] = []
```

Comment:
This file is general clean up and the adding of the emojis to the error messages